test/functional/fixtures/api/es-next/click/test.js

```diff
@@ -64,4 +65,23 @@ describe('[API] t.click()', function () {
                 );
             });
     });
+
+    it('Should not process the same driver status twice', function () {
```

Comment:
BTW, I'm not sure I understand how this test works. Can you describe it in the comment, please?